priv/templates/bin

```diff
@@ -69,16 +69,25 @@ ERTS_LIB_DIR="$ERTS_DIR/../lib"
 [ -f "$REL_DIR/$REL_NAME.boot" ] && BOOTFILE="$REL_NAME" || BOOTFILE=start
 cd "$ROOTDIR"
 
+{{! Define alternative field separator using ASCII US (unit separator). US is
+    not expected to appear in any argument. }}
```

Comment:
Used a [mustache comment](https://mustache.github.io/mustache.5.html) instead of a shell comment in order to keep it from appearing in the expanded bin script.